www/css/timeline.css

```diff
@@ -0,0 +1,226 @@
+/* -------------------------------- 
```

Comment:
Can this be moved to like `www/lib/timeline` just so it's obvious this isn't our code?